.github/workflows/sites-test.yml

```diff
@@ -0,0 +1,250 @@
+        options: '--shm-size="2g"'
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: drupal
+          MYSQL_USER: drupal
+          MYSQL_PASSWORD: drupal
+          MYSQL_ROOT_PASSWORD: drupal
+        ports:
+          - 33306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - uses: actions/checkout@v4
+      - name: Restore Cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            vendor
+            docroot/core
+            docroot/libraries
+            docroot/modules/contrib
+          key: 1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-${{ hashFiles('composer.lock') }}
+          restore-keys: |
+            1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-${{ hashFiles('composer.lock') }}
+            1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-
+            1.x-${{ hashFiles('blt/blt.yml') }}-
+      - run: git config --system --add safe.directory '*'
+      - name: Install Site
+        run: |
+          mysql -h mysql -P 3306 -u root -pdrupal -e 'SET GLOBAL max_allowed_packet=67108864;' &&
+          rm -rf /var/www/html &&
+          ln -snf $GITHUB_WORKSPACE /var/www/html &&
+          composer install -n &&
+          vendor/bin/blt blt:telemetry:disable --no-interaction &&
+          blt settings &&
+          mkdir -p docroot/sites/default/files &&
+          chmod -R 777 docroot/sites/default/files/ &&
+          blt drupal:install -n
+      - name: Codeception Tests
+        run: |
+          apachectl stop
+          apachectl start
+          blt codeception --group=install
+  deploy_branch:
+    needs:
+      - test_phpunit
+      - set_sites
+      - test_existing_site
+      - test_new_site
+    runs-on: ubuntu-latest
+    container:
+      image: pookmish/drupal8ci:latest
+      options: '--network-alias drupal8ci'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Restore Cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            vendor
+            docroot/core
+            docroot/libraries
+            docroot/modules/contrib
+          key: 1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-${{ hashFiles('composer.lock') }}
+          restore-keys: |
+            1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-${{ hashFiles('composer.lock') }}
+            1.x-${{ hashFiles('blt/blt.yml') }}-${{ hashFiles('composer.json') }}-
+            1.x-${{ hashFiles('blt/blt.yml') }}-
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          name: id_rsa
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+          if_key_exists: fail
+      - run: git config --system --add safe.directory '*'
+      - name: Deploy Branch
```

Comment:
Is this going to deploy every PR branch to Acquia, and not just the release branches? (And is that a problem)?

Reply:
yes. It's what we've been doing for a long time. I have a process that deletes unused branches in Acquia: https://github.com/SU-HSDO/suhumsci/pull/1481/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R85

Reply:
Oh wow. I need that for SDSS lol. We have way too many branches on Acquia.